Aspire Budgeting/Facilities/GoogleSheetsManager.swift

```diff
@@ -133,4 +186,13 @@ final class GoogleSheetsManager: ObservableObject {
       }
     }
   }
+  
+  func addTransaction(amount: String, date: Date, category: Int, account: Int, transactionType: Int, approvalType: Int) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/7eff2f7d46329351cc2fa8ce48ca6f8f1587b2ca/Aspire Budgeting/Facilities/GoogleSheetsManager.swift#L190"></span>Function should have 5 parameters or less: it currently has 6<br />
<code>function_parameter_count</code> <code>GoogleSheetsManager.swift:190</code></td>
    </tr>
  </tbody>
</table>